DXMainClient/Domain/Multiplayer/CnCNet/CnCNetPlayerCountTask.cs

```diff
@@ -86,6 +82,8 @@ private static int GetCnCNetPlayerCount()
                 return -1;
             }
         }
+
+        public static int PlayerCount { get; set; }
```

Comment:
Move this to the beginning of the class (like the other fields) and make the setter private.